modules/setting/setting.go

```diff
@@ -597,16 +598,16 @@ func NewContext() {
 		}
 	case "fcgi":
 		Protocol = FCGI
-	case "fcgi+unix":
-		Protocol = FCGIUnix
-		UnixSocketPermissionRaw := sec.Key("UNIX_SOCKET_PERMISSION").MustString("666")
-		UnixSocketPermissionParsed, err := strconv.ParseUint(UnixSocketPermissionRaw, 8, 32)
-		if err != nil || UnixSocketPermissionParsed > 0777 {
-			log.Fatal("Failed to parse unixSocketPermission: %s", UnixSocketPermissionRaw)
+	case "fcgi+unix", "unix", "http+unix":
+		switch protocolCfg {
+		case "fcgi+unix":
+			Protocol = FCGIUnix
+		case "unix":
+			Protocol = HTTPUnix
```

Comment:
A very tiny nitpick: You could use fallthrough after the log instead of this line.

Reply:
Oh, you're right, that's better. I am still a noob in Go, so I didn't knew that was a option.

Reply:
No worries:) Glad you were able to learn a new thing about go. Maybe more PRs are in your future ;)